src/main/java/com/snowflake/kafka/connector/Utils.java

```diff
@@ -393,12 +435,12 @@ static String validateConfig(Map<String, String> config) {
                 "Schematization is only available with {}.",
                 IngestionMethodConfig.SNOWPIPE_STREAMING.toString()));
       }
-      if (config.containsKey(SnowflakeSinkConnectorConfig.SNOWPIPE_STREAMING_FILE_VERSION)) {
+      if (config.containsKey(SnowflakeSinkConnectorConfig.SNOWPIPE_STREAMING_MAX_CLIENT_LAG)) {
         invalidConfigParams.put(
-            SnowflakeSinkConnectorConfig.SNOWPIPE_STREAMING_FILE_VERSION,
+            SnowflakeSinkConnectorConfig.SNOWPIPE_STREAMING_MAX_CLIENT_LAG,
             Utils.formatString(
                 "{} is only available with ingestion type: {}.",
-                SnowflakeSinkConnectorConfig.SNOWPIPE_STREAMING_FILE_VERSION,
+                SnowflakeSinkConnectorConfig.SNOWPIPE_STREAMING_MAX_CLIENT_LAG,
                 IngestionMethodConfig.SNOWPIPE_STREAMING.toString()));
       }
```

Comment:
@EduardHantig are you sure the `SNOWPIPE_STREAMING_FILE_VERSION` is replaced by `SNOWPIPE_STREAMING_MAX_CLIENT_LAG`?

Reply:
@JonathonO I have another PR where all changes are just pulled without solving conflicts. You can check here is the same also - https://github.com/streamkap-com/snowflake-kafka-connector/pull/16

Meanwhile I also checked git annotation of git blame:
<img width="1044" alt="Screenshot 2024-02-01 at 14 17 14" src="https://github.com/streamkap-com/snowflake-kafka-connector/assets/22726897/05816bb7-c21d-45f2-ab39-c74de1f0de2f">